onward/app/feed/MostViewedGalleryAgent.scala

```diff
@@ -32,7 +27,7 @@ object MostViewedGalleryAgent extends Logging with ExecutionContexts {
         url <- (item \ "url").asOpt[String]
         count <- (item \ "count").asOpt[Int]
       } yield {
-        getResponse(LiveContentApi.item(UrlToContentPath(url), Edition.defaultEdition)).map(_.content.map(Content(_)))
```

Comment:
This was actually causing a problem with the Video version of this. I fixed it earlier this week, but was obviously too lazy to look around and do the others.